docs/source/depth_estimation.mdx

```diff
@@ -29,7 +29,7 @@ Load the `train` split of the dataset and take a look at an example:
 >>> example = train_dataset[index]
 >>> example
 {'image': <PIL.PngImagePlugin.PngImageFile image mode=RGB size=640x480>,
- 'depth_map': <PIL.PngImagePlugin.PngImageFile image mode=L size=640x480>}
+ 'depth_map': <PIL.TiffImagePlugin.TiffImageFile image mode=F size=640x480>}
```

Comment:
This is new type which we should consider explaining.

Reply:
Simple explanation: "When it comes to storing data, JPEG and PNG are quite picky as they accept 8-bit or 16-bit unsigned integer data. However, However, TIFF is more flexible and supports a wider range of data types, including 32-bit floating point data. So if you need to store float32 data, TIFF is our best bet."

Reply:
Sounds good to me! 

@lhoestq WDYT?

Reply:
Sounds good to me as well :)